kratos/utilities/variable_utils.cpp

```diff
@@ -218,15 +210,11 @@ void VariableUtils::UpdateCurrentPosition(
 {
     KRATOS_TRY;
 
-    const int num_nodes = static_cast<int>(rNodes.size());
-    const auto it_node_begin = rNodes.begin();
-
-    #pragma omp parallel for
-    for (int i_node = 0; i_node < num_nodes; ++i_node) {
-        const auto it_node  = it_node_begin + i_node;
-        const auto& r_update_coords = it_node->FastGetSolutionStepValue(rUpdateVariable, BufferPosition);
-        noalias(it_node->Coordinates()) = (it_node->GetInitialPosition()).Coordinates() + r_update_coords;
-    }
+    auto rNodes_local = rNodes;
```

Comment:
```suggestion
    auto& rNodes_local = rNodes;
```
Pointers are automatic, references not

Reply:
Except if this is a copy...